pom.xml

```diff
@@ -33,6 +40,22 @@
             <artifactId>commons-lang</artifactId>
             <version>2.6</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-configuration2</artifactId>
+            <version>2.4</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-beanutils</groupId>
```

Comment:
Are we using this ?

Reply:
Yes. If it is not included you get an `NoClassDefFoundError: org/apache/commons/beanutils/DynaBean` exception when running the tests.

Reply:
But that's pretty strange isn't it ? Is it needed by the common config thing ?

Reply:
That is correct.